src/Entities/PaymentPlanTrait.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Alma\API\Entities;
+
+/**
+ * Trait PaymentPlanTrait
+ *
+ * @package Alma\API\Entities
+ */
+trait PaymentPlanTrait
+{
+
+    /**
+     * @return string
+     */
+    public function getPlanKey()
```

Comment:
Did we must validate integrity of the keys and return a proper string or false in place of "-_-_-_- " ?

Reply:
we want a string return.

`-_-_-_-` is returned only if entity is not set and attributes are `null` (avoid a _null string cast exception_ on _empty entity_)

if test on entity integrity have to be set, it should be done into `__construct()` where attributes are set, not into the getter 😉